engine/Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ["Chainflip team <https://github.com/chainflip-io>"]
 build = 'build.rs'
 edition = '2021'
 name = "chainflip-engine"
-version = '1.1.4'
```

Comment:
Is this what we are intending? @dandanlen @kylezs

Reply:
I thought we change all the version numbers together?

Reply:
I was trying out how to make it work! not sure what's the correct way here (first time trying out this new version bump thing).
Should I update all the versions of all our packages? lp-api, cli, etc.?

Reply:
We do always bump all versions at the same time. However in this case i think you can bump them all to `1.2.0`. The next release is going to be a breaking one. There are already some PRs waiting to be merged that bump us to 1.2.0, so it will be in line with that.

Reply:
In fact, since the version bump thing is not blocking, feel free to just remove the version bump - it will be added in another PR.

Reply:
Ok! reverted back the version bump!